files/bootstrap.sh

```diff
@@ -565,8 +572,12 @@ if [[ "$CONTAINER_RUNTIME" = "containerd" ]]; then
   sudo sed -i s,SANDBOX_IMAGE,$PAUSE_CONTAINER,g /etc/eks/containerd/containerd-config.toml
 
   echo "$(jq '.cgroupDriver="systemd"' "${KUBELET_CONFIG}")" > "${KUBELET_CONFIG}"
-  echo "$(jq '.systemReservedCgroup="/system"' "${KUBELET_CONFIG}")" > "${KUBELET_CONFIG}"
-  echo "$(jq '.kubeReservedCgroup="/runtime"' "${KUBELET_CONFIG}")" > "${KUBELET_CONFIG}"
+  ##allow --reserved-cpus options via kubelet arg directly and not set default cgroup option
+  if [[ "${USE_RESERVED_CPUS}" = false ]]; then
+    log "INFO: skipping setting of .systemReservedCgroup and .kubeReservedCgroup!"
```

Comment:
This log line is incorrect; within this `if` we *are* setting the cgroups. You already state that it'll be skipped in the other log line, so just remove this